Part3_InterruptRoutingSetupInAChipset.html

```diff
@@ -3,24 +3,24 @@
 <html lang="en">
 <head>
 	<meta charset="utf-8">
-	<title>External Interrupts in the x86 system. Part 3. Interrupt routing setup in a chipset on the example of Coreboot</title>
+	<title>External Interrupts in the x86 system. Part 3. Interrupt routing setup in a chipset using coreboot</title>
```

Comment:
The point of this article is to show how "interrupt routing setup in a chipset" is made, not how to do it in coreboot. It seems like the emphasis in your variant has shifted to the latter.
Coreboot in this article is just an example to show some concepts in the "External Interrupts in the x86 system" discussion.
If the headline need changes, could it be something different?

Reply:
Ok how about "Interrupt routing setup in a chipset, with the example of coreboot"?